scripts/dev-template.mjs

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env zx
+
+import fs from 'fs/promises'
+import path from 'path'
+
+const template = process.argv[3]
+const templatePath = `packages/cli/templates/${template}`
+const newConfigPath = `${templatePath}/node_modules/.${template}-vite.config.ts`
```

Comment:
maybe here we also need to make a small check if node_modules is present otherwise the app might crash in the next command?

Reply:
good point! but thinking of this.. if we didn't run `yarn install` yet, then `vite` is not there, and anyway nothing will work.